Gemfile

```diff
@@ -5,4 +5,4 @@ source "https://rubygems.org" do
 end
 
 
-gem "fastlane"
\ No newline at end of file
+gem 'fastlane', "2.107.0"
```

Comment:
Specifying a version here shouldn't be needed to pin `fastlane`'s version.

It is already done in `Gemfile.lock`, so `bundle install` will always install that version and running it with `bundle exec fastlane` should run the correct version.

@jkmassel Do you have a preference for specifying the version here as well as in `Gemfile.lock`? Perhaps something less specific like `gem "fastlane", "~> 2"` would be a good idea.

Reply:
I agree that it's not _necessary_, but if I run `bundle update`, it'll update it on me (which it should, because I invoked `update`, but now I'm using a different version from the team).

The big thing was that it seems like there was a breaking change in fastlane, which necessitated  the change in https://github.com/wordpress-mobile/WordPress-Android/commit/af3a56673b88ec4c31b67f0d83a0d788ab9fe16d – adding this in requires us to be a little more intentional about which precise version we're using.

Typically using a major version number _would_ be sufficient, but I'm hoping by being really specific we can avoid headaches down the road 😅

wdyt?